### DISCUSSION TITLE
Login issues after update to 2.8.0

### DISCUSSION THREAD

**Discussion by SMCerina** [automatic-ripping-machine/automatic-ripping-machine#1184] [Q&A]
After the update to 2.8.0 DISABLE_LOGIN: true is not working and unfortunately, I don't know the admin password. 
I'm using ARM in docker environment. The Docker log looks normal. If I could provide any further information let me know.

Is there any way to reset the password without losing the database?

I have downgraded to 2.7.2 again and DISABLE_LOGIN is working as intended and I’m able to use the software again.

**Comment by microtechno9000** [comment]
Had a look, its a bug. ARM isn't meant to run the check when login is disabled. Will see what I can do and raise a bugfix

**Comment by microtechno9000** [comment]
PR Raised - https://github.com/automatic-ripping-machine/automatic-ripping-machine/pull/1187
